Sources/Orbit/Foundation/Icons/Icons.swift

```diff
@@ -1,277 +1,542 @@
-/// Generated and updated by 'Automation/orbit/update_icons.py'
-// swiftlint:disable:previous orphaned_doc_comment
 
+// Generated and updated by 'Automation/update_icons.py'
 public extension Icon {
 
     enum Symbol: Comparable, CaseIterable {
```

Comment:
I know it's not new, but what is the `Comparable` conformance for?

Reply:
Removed, not needed (can be later provided by caller)